executor/aggfuncs/func_value.go

```diff
@@ -114,10 +138,9 @@ type value4String struct {
 	isNull bool
 }
 
-func (v *value4String) evaluateRow(ctx sessionctx.Context, expr expression.Expression, row chunk.Row) error {
-	var err error
+func (v *value4String) evaluateRow(ctx sessionctx.Context, expr expression.Expression, row chunk.Row) (memDelta int64, err error) {
 	v.val, v.isNull, err = expr.EvalString(ctx, row)
-	return err
+	return int64(unsafe.Sizeof(v.val)), err
```

Comment:
using sizeof to a string will return the memory usage of the string struct not the the memory usage of the buff of string.
see https://play.studygolang.com/p/youMqPAfT53

I think we should use len here.

Reply:
Ah, I see. Sorry for such mistakes. I'll fix it ASAP.
Seems like it's a long tour to go to learn golang ;)

Reply:
^_^ take it easy.